content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -17,7 +17,13 @@ plane and the sidecars for the Istio data plane.
 
 1. [Download the Istio release](/docs/setup/kubernetes/download-release/).
 
-1. [Kubernetes platform setup](/docs/setup/kubernetes/platform-setup/).
+1. Kubernetes platform setup
+  * [Minikube](/docs/setup/kubernetes/minikube/)
```

Comment:
Could we make the `Kubernetes platform setup` as the parent path and the platform setups as the sub-path?  Then it only links to the `Kubernetes platform setup` directory to avoid putting so many duplicated in their references.

Reply:
@lillywu I addressed this in PR#https://github.com/istio/istio.github.io/pull/1960

Reply:
Thanks. It looks great.